app/models/customer.rb

```diff
@@ -5,4 +5,9 @@ class Customer < ActiveRecord::Base
   has_many :pets, dependent: :destroy
   validates :name, presence: true,
                    length: { minimum: 2 }
+
+  validates_format_of :name, with: /\A\p{L}+(?:[-\s]\p{L}+)*\z/,
+                             message: 'must only contain letters, spaces and hyphens'
+  validates_format_of :phone, with: /\A[0-9]*\z/,
```

Comment:
Careful with `*` in regexp. Think about the difference between `+` and `*`

Reply:
Ok, my mistake.